lib/tether/tether_linux.go

```diff
@@ -140,9 +140,8 @@ func (t *tether) stopReaper() {
 	signal.Reset(syscall.SIGCHLD)
 
 	// just closing the incoming channel is not going to stop the iteration
-	// so we use done channel to signal it
-	log.Debugf("Signalling the child reaper loop")
-	close(t.done)
+	// so we use the context cancellation to signal it
+	t.cancel()
```

Comment:
Agreed but right now we are stopping them only when tether is done and ctx seems to be doing that well. We can add more granular control to tether but to be honest I prefer not doing it here but a separate PR (maybe with a big tether refactoring)